verifiable-db/src/revelation/revelation_unproven_offset.rs

```diff
@@ -60,16 +60,65 @@ use crate::{
             build_cells_tree,
             universal_circuit_inputs::{BasicOperation, Placeholders, ResultStructure},
         },
-        PI_LEN,
+        PI_LEN as QUERY_PI_LEN,
     },
 };
 
 use super::{
     placeholders_check::{CheckPlaceholderGadget, CheckPlaceholderInputWires},
     revelation_without_results_tree::CircuitBuilderParams,
-    PublicInputs, NUM_PREPROCESSING_IO, NUM_QUERY_IO, PI_LEN as REVELATION_PI_LEN,
+    PublicInputs, NUM_PREPROCESSING_IO, PI_LEN as REVELATION_PI_LEN,
 };
 
+#[derive(Clone, Debug, Serialize, Deserialize, Default, PartialEq, Eq)]
+/// Data structure containing all the information needed to verify the membership of
+/// a row in a tree representing a table
+pub struct RowPath {
```

Comment:
There is a already a `RowPath` defined - linked to my comments about name conflicts between vTable row and row tree.

Reply:
And it looks to me it contains _exactly_ the same information as the `RowPath` defined in `batching/circuits/api`  - what am i missing ?

Reply:
They are not the same, this one doesn't need the data about the children of the node, which is instead necessary in the one employed in batching circuits API